content/en/docs/concepts/services-networking/service.md

```diff
@@ -555,7 +555,7 @@ status:
       - ip: 146.148.47.155
 ```
 
-Traffic from the external load balancer is directed at the backend Pods. The cloud provider decides how it is load balanced.
+Traffic from the external load balancer is directed at the backend Pods. The cloud provider decides how it is load balanced. On baremetal clusters you can use an already provisioned appliance such as [Citrix ADC](https://developer-docs.citrix.com/projects/citrix-k8s-ingress-controller/en/latest/network/type_loadbalancer/) to provide support for `type LoadBalancer`. Another solution for baremetal clusters is to use the [MetalLB](https://metallb.universe.tf) project.
```

Comment:
@chiradeep This page is already very long. I'm wary of making it longer.

Although it's more work, what I think would better serve readers is a different change: add a new Task page about exposing services in bare metal clusters, and then include only the small amount of text needed to hyperlink there.

If there is an existing Task page that someone can adapt, that's fine too.

Most of this is my opinion and not an outright rejection. It's definitely a fact that the Service concept page is really long, though.

Reply:
Feature docs aren't vendor ad space.

Reply:
it isn't vendor ad space. Unless you consider all the GPC/AWS/Azure stuff as non-vendor